src/SDKs/PostgreSQL/AzSdk.RP.props

```diff
@@ -0,0 +1,7 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
+  <PropertyGroup>
+    <AzureApiTag>PostgreSQL_2017-12-01;</AzureApiTag>
```

Comment:
@rohit-joy please execute msbuild build.proj /t:Build /p:Scope=SDKs\PostgreSQL
your API tag should match your real RP namespace.

Reply:
Done